webvtt.html

```diff
@@ -542,6 +542,17 @@
    </section>
 
    <section>
+    <h3>Unicode normalization</h3>
+
+    <p>This specification does not normalize Unicode text as part of its processing
+    requirements.</p>
+
+    <p>For example, a cue with the identifier consisting of the LATIN CAPITAL LETTER A followed by
```

Comment:
I am unclear.  I get that the third of these is stated not to be the case, but I still have to choose between the other two:

* I am required not not normalize
* I am not required to normalize, but I may
* I am required to normalize

…

?

> On Jun 8, 2015, at 7:15 , Philip Jägenstedt <notifications@github.com> wrote:
> 
> In webvtt.html:
> 
> > @@ -542,6 +542,17 @@
> >     </section>
> >  
> >     <section>
> > +    <h3>Unicode normalization</h3>
> > +
> > +    <p>This specification does not normalize Unicode text as part of its processing
> > +    requirements.</p>
> > +
> > +    <p>For example, a cue with the identifier consisting of the LATIN CAPITAL LETTER A followed by
> 
> Maybe "consisting of the characters U+0041 LATIN CAPITAL LETTER A and U+212B ANGSTROM SIGN"?
> 
> —
> Reply to this email directly or view it on GitHub.
> 

David Singer
Manager, Software Standards, Apple Inc.